### PR TITLE
fix #69: move code into src/ for web-ext build

### LINKED ISSUE #69: Audit files in XPI
Unzipping the **/web-ext-artifacts/blok-0.1.7-fx.xpi**, there are a few interesting results:

```sh
$ unzip blok-0.1.7-fx.xpi
$ tree

├── META-INF/
│   ├── manifest.mf
│   ├── mozilla.rsa
│   └── mozilla.sf
├── README.md
├── coverage/ (EMPTY)
├── css/
│   ├── contentscript.css
│   ├── feedback.css
│   ├── foundation.min.css
│   └── toolbar.css
├── disconnect-blocklist.json
├── disconnect-entitylist.json
├── feedback.html
├── icons/ (EMPTY)
├── img /
│   ├── banner.png
│   ├── blok.png
│   ├── screenshot.png
│   ├── tracking-protection-16.svg
│   └── tracking-protection-disabled-16.svg
├── js/
│   ├── background.bundle.js
│   ├── background.js
│   ├── canonicalize.js
│   ├── contentscript.js
│   ├── feedback.js
│   ├── lists.js
│   ├── log.js
│   └── toolbar.js
├── manifest.json
├── package.json
├── tests/
│   ├── blocklist-fixture.json
│   ├── test-canonicalize.js
│   └── test-lists.js
├── toolbar.html
└── web-ext-artifacts/ (EMPTY)

8 directories, 30 files
```

```sh
$ ls -lh

drwxr-xr-x   5 pdehaan  staff   170B Jul 28 16:28 META-INF
-rw-r--r--   1 pdehaan  staff   1.6K Jul 27 14:48 README.md
drwxr-xr-x   2 pdehaan  staff    68B Jul 27 14:48 coverage/
drwxr-xr-x   6 pdehaan  staff   204B Jul 27 14:48 css/
-rw-r--r--   1 pdehaan  staff   260K Jul 27 14:48 disconnect-blocklist.json
-rw-r--r--   1 pdehaan  staff   196K Jul 27 14:48 disconnect-entitylist.json
-rw-r--r--   1 pdehaan  staff   1.7K Jul 27 14:48 feedback.html
drwxr-xr-x   2 pdehaan  staff    68B Jul 27 14:48 icons/
drwxr-xr-x   7 pdehaan  staff   238B Jul 27 14:48 img/
drwxr-xr-x  10 pdehaan  staff   340B Jul 27 14:48 js/
-rw-r--r--   1 pdehaan  staff   719B Jul 27 14:48 manifest.json
-rw-r--r--   1 pdehaan  staff   1.6K Jul 27 14:48 package.json
drwxr-xr-x   5 pdehaan  staff   170B Jul 27 14:48 tests/
-rw-r--r--   1 pdehaan  staff   1.7K Jul 27 14:48 toolbar.html
drwxr-xr-x   2 pdehaan  staff    68B Jul 27 14:48 web-ext-artifacts/
```

Most curious are the empty `/coverage/`, `/icons/` directories (/icons/ was renamed to /img/ in #63), and `/web-ext-artifacts/`.

I'm guessing now that we've merged #43 we'll also have a /docs/ directory in the next build.

We _probably_ (maybe?) can remove the following things from the generated XPI add-on:
- /coverage/
- /docs/*
- /icons/
- /tests/*
- /web-ext-artifacts/
- /img/*.png (add-on may only need *.svg files)
- /js/background.js (**?** since we have a browserified /js/background.bundle.js)
- /README.md
- /manifest.json (**?** not sure if needed in the .XPI or just to build the XPI)
- /package.json (**?** not sure if needed in the .XPI or just to build the XPI)

### PR DESCRIPTION
**Pull request by groovecoder**
Moving the extension source files into `src/` and using `web-ext build -s src` trims out about 600K from the file-size:
```
-rw-r--r--  1 lcrouch  staff   753K Sep  8 13:52 blok-1.0rc1.xpi
-rw-r--r--  1 lcrouch  staff   157K Sep  8 14:06 blok-1.0rc2.xpi
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -39,16 +39,16 @@ When the add-on blocks tracker requests, you will see a Tracking Protection page
 
 ### With `web-ext`
 
-1. [Install `web-ext`](https://github.com/mozilla/web-ext/#documentation) if
-   you haven't already
-2. `web-ext run --firefox-binary {path to Firefox 49+ binary}`
+1. `npm install`
+2. `cd src`
+3. `../node_modules/.bin/web-ext run --firefox-binary {path to Firefox 49+ binary}`
```
Comment (pdehaan) [inline README.md]: I wonder if adding the web-ext as a npm "script" in package.json would be clearer.
For example, running `$ web-ext build --help` gives me a `--source-dir={}` option, which may help avoid your `cd src && ../node_modules/.bin/web-ext run ...` speedbump.

For example, you'd be able to run `npm run build` or `npm run build -- --firefox-binary {path to Firefox 49+ binary}` (if you want to pass args to it).

```sh
$ ./node_modules/.bin/web-ext build --help
node_modules/.bin/web-ext build

Options:
  --version                   Show version number                                                                                   [boolean]
  --source-dir, -s            Web extension source directory.
                                                         [string] [required] [default: "/Users/pdehaan/dev/github/mozilla-services/pageshot"]
  --artifacts-dir, -a         Directory where artifacts will be saved.
                                       [string] [required] [default: "/Users/pdehaan/dev/github/mozilla-services/pageshot/web-ext-artifacts"]
  --verbose, -v               Show verbose output                                                                        [boolean] [required]
  --as-needed                 Watch for file changes and re-build as needed                                                         [boolean]
  -h, --help, --help, --help  Show help
```

**Comment by pdehaan** [comment]
r+ w/ nit above.

**Comment by groovecoder** [comment]
Changed the run command (back) to a single line: `./node_modules/.bin/web-ext run --source-dir=src --firefox-binary {path to Firefox 49+ binary}`

**Comment by pdehaan** [comment]
For my own future reference, here's what the XPI currently looks like today:

blok-1_0rc2_xpi

**Pull request merged**